web/solutions/stripe-payments.md

```diff
@@ -0,0 +1,100 @@
+# Stripe Payment Integration
+
+* Preparation: **2 minute**
+* Requirements:
+  * Initiated Syncano project
+  * Stripe secret_key
+  * Stripe publishable_key
+
+### Problem to solve
+
+You want to integrate payment platform to your web application that allows users make payments using credit cards such as:
+
+* Visa
+* Visa (debit)
+* Mastercard
+* Mastercard (2-series)
+* Mastercard (debit)
+* Mastercard (prepaid)
+* American Express
+* American Express
+* Discover
+* Discover
+* Diners Club
+* JCB
+
+### Solution
+
+Our solution is establised using [stripe-payments](https://syncano.io/#/sockets/stripe-payments) to trigger payment service.
+
+### Installing dependencies
+
+#### Server-side
+
+To install stripe-payments, type:
+
+```sh
+$ syncano-cli add stripe-payments
+```
+
+Provide `STRIPE_SECRET_KEY` and `STRIPE_PUBLISHABLE_KEY`
+
+* N/B: To find the keys, log into your Stripe account, navigate to [API](https://dashboard.stripe.com/account/apikeys).
+
+Deploy stripe-payment to update socket
+
+```sh
+$ syncano-cli deploy stripe-payments
+```
+
+#### Client-side
+
+Install syncano-client to interact with Syncano stripe-payments socket:
+
+```sh
+$ npm install syncano-client --save
+```
+
+or
+
+```HTML
+<script src="https://unpkg.com/syncano-client"></script>
+```
+
+To use Syncano library, import:
+
+```javascript
+import Syncano from "syncano-client";
```

Comment:
This should be better phrased here. 
`<script src="https://unpkg.com/syncano-client"></script>`
is the html version of 
`import Syncano from "syncano-client";`
which is used in es6 js apps.

we should make it clear these are two way's of achieving the same thing.

Reply:
done

Reply:
Not what I meant. 

1. When using webpack and es6 the way to handle the client lib is:
Shell:
```sh
npm i -S @syncano/client
```

Code:
```js
import SyncanoClient from "@syncano/client"

const s = new SyncanoClient("MY_INSTANCE_NAME");
```

2. And the vanilla js way:
```html
<script src="https://unpkg.com/@syncano/client"></script>
<script>
  const s = new SyncanoClient("MY_INSTANCE_NAME");
</script>
```